includes/Integrations/Core_Themes_Support.php

```diff
@@ -77,7 +105,29 @@ public function extend_theme_support() {
 	 */
 	public function embed_web_stories() {
 		$customizer = new Customizer();
-		echo $customizer->render_stories(); // phpcs:ignore -- WordPress.Security.EscapeOutput.OutputNotEscaped - Escaped web stories HTML.
+		$this->enqueue_style( $this->style_handle );
+		?>
+		<div class="web-stories-theme-header-section">
+			<?php echo $customizer->render_stories(); // phpcs:ignore -- WordPress.Security.EscapeOutput.OutputNotEscaped - Escaped web stories HTML. ?>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Add a class if it is one of supported core themes.
+	 *
+	 * @since 1.3.0
+	 * @param array $classes Array of body classes.
+	 *
+	 * @return array Updated array of classes.
+	 */
+	public function add_core_theme_classes( $classes ) {
+
+		if ( $this->showing_customizer_stories ) {
```

Comment:
Don't need this check because there's already a check in `init()` below

Reply:
Yes, thanks to point it out, was required previously but not now.